base/strings/basic.jl

```diff
@@ -585,16 +604,19 @@ julia> ucfirst("python")
 ```
 """
 function ucfirst(s::AbstractString)
-    isempty(s) && return s
+    isempty(s) && return ""
     c = s[1]
-    tc = titlecase(c)
-    return c==tc ? s : string(tc,s[nextind(s,1):end])
+    c′ = titlecase(c)
+    c == c′ ? convert(String, s) :
+    string(c′, SubString(s, nextind(s, 1)))
```

Comment:
Fix indentation.

Reply:
Do you prefer the "else" expression of a ternary to be indented or not – the above comment seems to indicate no but this comment seems to indicate yes...